pegasus/migrations/75_add_created_at_column_to_storage_apps.rb

```diff
@@ -0,0 +1,32 @@
+# Migration for extracting JSON.parse(row[:value])['createdAt'] into a :created_at column.
+Sequel.migration do
+  up do
+    add_column :storage_apps, :created_at, DateTime
+
+    batch_update do |row|
+      value = JSON.parse(row[:value])
+      created = DateTime.parse(value['createdAt']) if value['createdAt']
+
+      # Set :created_at.
+      from(:storage_apps).where(id: row[:id]).update(created_at: created)
+    end
+  end
+
+  down do
+    drop_column :storage_apps, :created_at
+  end
+end
+
+# Update rows in batches.
+def batch_update
+  offset = 0
+  batch_size = 1000
+  loop do
+    batch = from(:storage_apps).where(state: 'active').offset(offset).limit(batch_size)
```

Comment:
@davidsbailey To confirm, we're only populating `:created_at` for active channels.  Is that the right thing to do?  I excluded rows with `state: deleted` because some of them may be unparsable due to the emoji bug we fixed.